json-endpoints/class.wpcom-json-api-comment-endpoint.php

```diff
@@ -88,12 +88,12 @@ function get_comment( $comment_id, $context ) {
 				&&
 					$this->api->token_details['user']['display_name'] === $comment->comment_author
 				) {
-					$user_can_read_coment = true;
+					$user_can_read_comment = true;
 				} else {
-					$user_can_read_coment = current_user_can( 'edit_comment', $comment->comment_ID );
+					$user_can_read_comment = current_user_can( 'edit_posts' );
 				}
 
-				if ( !$user_can_read_coment ) {
+				if ( !$user_can_read_comment ) {
```

Comment:
Since you're editing that line, could you add a space after the `!` to make it super clean?